clients/web/src/templates/widgets/itemList.jade

```diff
@@ -3,9 +3,13 @@ ul.g-item-list
     li.g-item-list-entry
       if checkboxes
         input.g-list-checkbox(type="checkbox", g-item-cid="#{item.cid}")
-      a.g-item-list-link(g-item-cid="#{item.cid}")
+      a.g-item-list-link.g-right-border(g-item-cid="#{item.cid}")
         i.icon-doc-text-inv
         | #{item.get('name')}
+      a.g-right-border(href=item.downloadUrl())
```

Comment:
I think we should remove the `g-right-border` class here so there's just one separator instead of two.